src/editorial/web/components/starRating/star.ts

```diff
@@ -0,0 +1,44 @@
+import { neutral } from '@guardian/src-foundations/palette';
+import { svgBackgroundImage } from '@guardian/src-helpers';
+
+const spacing = 15;
```

Comment:
Would it break things terribly if we used 16px spacing instead of 15, to fit with our [space scale](https://theguardian.design/2a1e5182b/p/449bd5-space)?

Reply:
It looked just a tad too spacey… but I’m not against it.


| 15px      | 16px      |
|-----------|-----------|
| ![15][]   | ![16][]   |

[15]: https://user-images.githubusercontent.com/76776/115030661-564af200-9e95-11eb-8d26-ae63ec4ad499.png
[16]: https://user-images.githubusercontent.com/76776/115030714-65ca3b00-9e95-11eb-8bcd-3c7dbe3a4347.png

Reply:
Fair point! I'll defer to an opinion from design, if anyone has one @HarryFischer @akemitakagi

Otherwise, let's leave it how it is 😎

Reply:
![designers-have-spoken](https://user-images.githubusercontent.com/76776/116521518-124aea80-a8a2-11eb-9df2-7b2e7e5426f0.png)